source/containers.tex

```diff
@@ -3877,8 +3887,9 @@
 \begin{itemdescr}
 \pnum
 \result
-\tcode{pair<iterator, iterator>};
-\tcode{pair<const_iterator, const_iterator>} for constant \tcode{b}.
+\tcode{pair<const_iterator, const_iterator>}
+if \tcode{b} is of type \tcode{const X}; \\
+\tcode{pair<iterator, iterator>}~otherwise.
```

Comment:
Please drop the `~` line-break prevention.

Instead of `\\`, can you add a hyphenation hint `\-` in a suitable position?

Reply:
Alright, I've cleaned up the commit message and gotten rid of the `~`.

I couldn't get the hyphenation hint to work. The issue is not that any word is too long and needs hyphenation; the issue is that TeX insists on putting `pair<iterator, iterator>` on the same line as the text preceding it. I have no idea why, but it wouldn't separate it.